app/src/main/java/org/fossasia/openevent/general/event/EventViewHolder.kt

```diff
@@ -28,7 +30,7 @@ class EventViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         event.originalImageUrl?.let {
             Picasso.get()
                     .load(it)
-                    .placeholder(R.drawable.ic_launcher_background)
+                    .placeholder(AppCompatResources.getDrawable(OpenEventGeneral.appContext!!, R.drawable.ic_launcher_background)!!)   //TODO: Make null safe
```

Comment:
appContext is being used

Reply:
But this does not extend fragment or activity; what other context should I use?

Reply:
You have itemView which has context